submissions/rubyAdeniregun/Database/init.sql

```diff
@@ -0,0 +1,161 @@
+CREATE DATABASE ecommerce_storedatabase;
+/* name of datebase */
+
+use ecommerce_storedatabase;
+
+/* shows the different product categories in the database */
+CREATE TABLE product_categories (
+id INT NOT NULL AUTO_INCREMENT,
+categories VARCHAR (100) NOT NULL,
+created_at datetime NOT NULL,
+primary key(id)
+);
+
+
+INSERT INTO product_categories (categories, created_at)
+VALUES 
+	('Skincare', now()),
+	('Haircare', now()),
+	('Cereals', now()),
+	('Fashion', now()),
+	('Electronics', now());
+    
+SELECT * FROM product_categories;
+
+/* shows the list of administrative staff that has access to the database */ 
+CREATE TABLE adminstaff_users (
+id INT NOT NULL AUTO_INCREMENT,
+name VARCHAR(100) NOT NULL,
+email VARCHAR(100) NOT NULL,
+password VARCHAR(20) NOT NULL,
+phonenumber INT NOT NULL,
+created_at datetime NOT NULL,
+primary key(id)
+);
+
+INSERT INTO adminstaff_users (name, email, password, phonenumber, created_at) 
+VALUES
+('Vera Thompson', 'verathompson@lilistores.com', 'ver2345', '0813776566', now()),
+('Lilian Bach', 'lilianbach@lilistores.com', 'lily6755', '0819456522', now()),
+('Howard Bark', 'howardbark@lilistores.com', 'hown0976', '0807347785', now());
+
+SELECT * FROM adminstaff_users;
+
+SHOW TABLES;
+
+/* shows the list of products */
+CREATE TABLE products (
+id BIGINT NOT NULL AUTO_INCREMENT,
+created_by_adminstaff_users_id INT NOT NULL, -- links products to the admin staff who uploaded the product
+product_categories_id INT NOT NULL, -- links products to categories
+product_name VARCHAR(100) NOT NULL,
+product_description VARCHAR(255) NOT NULL,
+product_image BLOB(255) NOT NULL,
+unit_price DECIMAL (20,2) NOT NULL,
+quantity_available INT NOT NULL,
+status VARCHAR(40) NOT NULL,
+creation_date DATE,
+created_at datetime NOT NULL,
+primary key (id),
+FOREIGN KEY (created_by_adminstaff_users_id) REFERENCES adminstaff_users (id),
+FOREIGN KEY (product_categories_id) REFERENCES product_categories (id)
+);
+
+INSERT INTO products 
+(created_by_adminstaff_users_id, product_categories_id, product_name, product_description, product_image, unit_price, quantity_available, status, creation_date, created_at) 
+VALUES
+('2', '3', 'Morning Fruity Granola', 'Give yourself a treat with our fruity granola every morning. Filled with nutrients and vitamins', 'granola.url', '4500.00', '15', 'in stock', cast(now() AS date), now()),
```

Comment:
the `id` and `product_categories_id` are `int`, but the values you are inserting are `string or varchar`. It should be `2, 3` not `'2', '3'`. Same goes for other places where you made this same mistake. You also made the mistake for `DECIMAL` too